core-services/prow/02_config/_plugins.yaml

```diff
@@ -243,6 +243,18 @@ bugzilla:
       target_release: 4.4.z
       validate_by_default: true
     release-3.11:
+      add_external_link: true
+      exclude_defaults: true
+      is_open: true
+      state_after_merge:
+        status: MODIFIED
+      state_after_validation:
+        status: POST
+      valid_states:
+      - status: NEW
+      - status: ASSIGNED
+      - status: ON_DEV
+      - status: POST
```

Comment:
This needs a `dependent_bug_target_releases: []` to solve the original problem. Are  all the other settings not inherited?
CC @AlexNPavel

Reply:
Happy to let dptp take over implementation of the intent. Feel free to close this with a better PR.

Reply:
I don't think it does - empty list would IMO mean that no bug would ever passed the validation, because that way the bot would check the dependent bugs, but would not consider any release as valid. Not having `d_b_t_r` means the check should not be done at all.

Reply:
okay, the code does indeed a `nil` check rather than a `len() > 0` check. @petr-muller so I guess we have to fix this upstream?

Reply:
@alvaroaleman possibly. Technically the code is right (`nil` means don't check, empty list means that no value is valid), but I cannot imagine a real use case for an empty set of valid values, and treating an empty list as "don't check" would give us the nice ability to disable the checks inherited from higher levels. I guess we would need to make all fields behave like this though?